packages/ui/components/editor/plugins/ToolbarPlugin.tsx

```diff
@@ -384,7 +384,7 @@ export default function ToolbarPlugin(props: TextEditorProps) {
       editor.dispatchCommand(TOGGLE_LINK_COMMAND, null);
     }
   }, [editor, isLink]);
-  return (
+  return props.editable ? (
```

Comment:
[Prefer Early Returns](https://handbook.cal.com/engineering/best-practices/prefer-early-returns) 

```suggestion
  if (!props.editable) return  <></>
```